### ISSUE TITLE
Is reference voltage relevant for the thermistor tables?

### ISSUE THREAD

**Issue by Traumflug** [Traumflug/Teacup_Firmware#180]
More a question instead of a bug. I just did the first steps of analog reading on ARM, which runs on 3.3V. I get the very same ADC readouts with the same thermistor as with the 5V-AVR.

Looking a bit closer, this isn't much of a surprise, because the circuitry is entirely passive, so one doesn't get an absolute voltage, but a fraction of the reference voltage. NXPs manual describes it like this:

> this field contains a binary fraction representing X the voltage on the ADn pin selected by the SEL field, divided by the voltage on the Vdd pin. Zero in the field indicates that the voltage on the ADn pin was less than, equal to, or close to that on Vss [ = GND] , while 0x3FF indicates that the voltage on ADn was close to, equal to, or greater than that on Vref.

I'm just not sure wether the math used for the thermistor tables sees this the same way. If yes, this math could be simplified and the voltage value removed from the Beta algorithm. Fewer fields -> easier to use.

The Steinhart-Hart equitation doesn't seem to take notice of the voltage anyways.

**Comment by triffid** [comment]
Yes, in the case of a resistor divider from ~avcc, the ADC reading is
entirely ratiometric and the voltage cancels out completely.

There should be no need to have supply voltage in the math with the usual
thermistor input circuitry.
